frontend/src/scenes/funnels/funnelLogic.ts

```diff
@@ -566,6 +673,16 @@ export const funnelLogic = kea<funnelLogicType>({
 
     listeners: ({ actions, values, props }) => ({
         loadResultsSuccess: async () => {
+            // set visibility of baseline or first five breakdowns for each step
+            values.steps?.forEach((step) => {
+                values.flattenedStepsByBreakdown
+                    ?.filter((s) => !!s.breakdown)
+                    ?.slice(5)
+                    .forEach((b) => {
+                        actions.setHiddenById({ [getVisibilityIndex(step, b.breakdown_value)]: true })
```

Comment:
The comment is slightly misleading, as this sets the visibility of everything except the first 5?

Reply:
got it! will update the comment